lighthouse-core/report/html/templates.html

```diff
@@ -617,7 +617,7 @@
       color: var(--body-text-color);
     }
 
-    /* TODO(#8185) use more BEM (.lh-gauge__label--big) instead of relying on descendent selector */
+    /* TODO(#8185) use more BEM (.lh-gauge__label--big) instead of relying on descendant selector */
```

Comment:
I stand by @Hoten on this one

Oxford says descendant === noun, descendent === adjective
https://en.oxforddictionaries.com/definition/descendant